tests/GenericExtractorJobTest.php

```diff
@@ -129,7 +129,7 @@ public function testFirstPage()
      * @param array $config
      * @param array $expectedParams
      */
-    public function testNextPage($config, $expectedParams)
+    public function testNextPage($config, $expectedParams): void
```

Comment:
`array $config,array $expectedParams` a můžeš to smazat nahoře z komentáře

Reply:
jj

Reply:
https://github.com/keboola/generic-extractor/pull/134/commits/6a5371ec44d089be4d460c4606d7278f5fd4b980